google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1alpha2/SchemaCompatibility.java

```diff
@@ -348,6 +343,9 @@ private void protoFieldTypeIsCompatibleWithBQFieldType(
       case "NUMERIC":
         match = isCompatibleWithBQNumeric(protoType);
         break;
+      case "BIGNUMERIC":
+        match = isCompatibleWithBQNumeric(protoType);
```

Comment:
please add test for this

Reply:
Removed it for now, it seems the value is not available in external library yet.